examples/spawning-processes/spawning-processes.go

```diff
@@ -30,16 +31,22 @@ func main() {
 
 	// `Output` and other methods of `Command` will return
 	// `*exec.Error` if there was a problem executing the
-	// command (e.g. wrong path), and `*exec.ExitError`
-	// if the command ran but exited with a non-zero return
-	// code.
+	// command (e.g. executable file not found in $PATH),
+	// and `*exec.ExitError` if the command ran but
+	// exited with a non-zero return code.
 	_, err = exec.Command("date", "-x").Output()
 	if err != nil {
-		switch e := err.(type) {
-		case *exec.Error:
+		var (
```

Comment:
I'd just do:

```
var execErr *exec.Error
var exitErr *exec.ExitError

```

IMHO saving one `var` keyword doesn't justify the two extra lines